### PR TITLE
Improves RSI Validator

### PR DESCRIPTION
**Pull request by ike709**
## About the PR 

I've forked the github action. It no longer errors when a `meta.json` is deleted, and also reduced some unnecessary logging.

In the future we can verify that PNGs actually exist if there's a state name for them, but for now I just want to fix the error.

**Screenshots**

N/A

**Changelog**

N/A

### CODE REVIEW

**Comment by ike709** [comment]
Fork: https://github.com/ike709/json-schema-validator

**Comment by PaulRitter** [comment]
forked to https://github.com/space-wizards/json-schema-validator, please update the pr to use that one instead

File: .github/workflows/validate-rsis.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: Parse files
-        uses: snapcart/json-schema-validator@v1.0.0
+        uses: ike709/json-schema-validator@v1.0.1
```
Comment (ike709) [inline .github/workflows/validate-rsis.yml:11]:
```suggestion
        uses: space-wizards/json-schema-validator@v1.0.2
```

**Comment by ike709** [comment]
@PaulRitter done

**Review by PaulRitter — APPROVED** [review]

**Pull request merged**